docs/builders/automode-builder.md

```diff
@@ -0,0 +1,88 @@
+# Auto Mode Builder
+
+The `AutomodeBuilder` allows you to get started with a builder class to configure with required setup as you prepare a blueprint for setting up EKS cluster with EKS Auto Mode.
+
+The `AutomodeBuilder` creates the following:
+
+- An EKS Cluster with passed k8s version and cluster tags.
+- The specified NodePools and EKS Auto Mode storage and networking configurations.
+
+## Input Parameters
+
+`Partial<AutomodeClusterProviderProps>` parameters can be used as inputs to `AutomodeBuilder`. Few parameters shown below:
+
+- `version` : Kubernetes version to use for the cluster
+- `nodePools`: The Auto Mode node pools to provision for the cluster
+
+### Demonstration - Running EKS Auto Mode
+
+The below usage helps you with a demonstration to use `AutomodeBuilder` to configure a required setup as you prepare a blueprint for setting up EKS Auto Mode on a new EKS cluster.
+
+```typescript
+import * as blueprints from "@aws-quickstart/eks-blueprints";
+import { AutomodeBuilder } from "@aws-quickstart/eks-blueprints";
+import { CfnWorkspace } from "aws-cdk-lib/aws-aps";
+import * as eks from "aws-cdk-lib/aws-eks";
+import { Construct } from "constructs";
+
+export default class AutomodeConstruct {
```

Comment:
minor: Unless you plan to move it as is to the patterns repo, I suggest we drop the outer AutomodeConstruct class. If you will add it to the patterns, where we use this convention, we can keep as is.

Reply:
OK, we can do after the release.